tests/shared/src/test/scala/spire/math/extras/interval/IntervalSeqSuite.scala

```diff
@@ -11,26 +11,27 @@ class IntervalSeqSuite extends munit.FunSuite {
     val b = atOrAbove(1)
     val c = point(1)
     val d = hole(1)
-    assert(atOrAbove(1) == (a | b))
-    assert(above(1) == (a & b))
-    assert(point(1) == (a ^ b))
-    assert(atOrAbove(1) == (a | c))
-    assert(empty[Int] == (a & c))
-    assert(atOrAbove(1) == (a ^ c))
-    assert(hole(1) == (a | d))
-    assert(above(1) == (a & d))
-    assert(below(1) == (a ^ d))
+    assertEquals(atOrAbove(1), (a | b))
+    assertEquals(above(1), (a & b))
+    assertEquals(point(1), (a ^ b))
+    assertEquals(atOrAbove(1), (a | c))
+    assertEquals(empty[Int], (a & c))
+    assertEquals(atOrAbove(1), (a ^ c))
+    assertEquals(hole(1), (a | d))
+    assertEquals(above(1), (a & d))
+    assertEquals(below(1), (a ^ d))
```

Comment:
What happened to `==`?

Reply:
Nothing, this is just a change to `munit` assertEquals

Reply:
Ah, thanks, I thought it was more syntax problems 👍